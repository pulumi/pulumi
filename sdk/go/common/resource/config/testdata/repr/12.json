{"my:mapKey":{"a":{"secure":"securevalue"},"c":"d"}}