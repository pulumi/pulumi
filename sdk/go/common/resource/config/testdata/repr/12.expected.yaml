decrypt:
  my:mapKey: '{"a":{"secure":"securevalue"},"c":"d"}'
paths:
  my:mapKey:
    value:
      a:
        secure: securevalue
      c: d
    string: '{"a":{"secure":"securevalue"},"c":"d"}'
    redacted: '{"a":"[secret]","c":"d"}'
    object: KBAAF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4EEAQL/ggABDAEQAABP/4JMAAIBYRdtYXBbc3RyaW5nXWludGVyZmFjZSB7ff+CHwABBnNlY3VyZQZzdHJpbmcMDQALc2VjdXJldmFsdWUBYwZzdHJpbmcMAwABZA==
    secure: true
    isObject: true
    secureValues:
    - securevalue
  my:mapKey.a:
    value:
      secure: securevalue
    string: securevalue
    redacted: '[secret]'
    object: GBAABnN0cmluZwwNAAtzZWN1cmV2YWx1ZQ==
    secure: true
    isObject: false
    secureValues:
    - securevalue
  my:mapKey.c:
    value: d
    string: d
    redacted: d
    object: DhAABnN0cmluZwwDAAFk
    secure: false
    isObject: false
